### DISCUSSION TITLE
mute subxsheet

### DISCUSSION THREAD

**Discussion by phil123456** [opentoonz/opentoonz#6730] [Ideas]
### Ask a question related to OpenToonz

Hi

don't know if it's a bug, but as I disable a sub sheet I'd like it to be quiet...(obviously?)
I tried both buttons but the audio still plays

![image](https://user-images.githubusercontent.com/13568652/200083620-862e55b0-33cf-41ed-921c-2013db4d1603.png)

would be nice not to have to open all subsheet to quiet them

by using a volume slider? really?
![image](https://user-images.githubusercontent.com/13568652/200084041-bfd7dfba-cf0a-4f4d-8419-04fe3c8f55c3.png)

oh wait there is more... **as I close the subsheet it plays full volume although I set it to zero**... what?

thanks

**Comment by RodneyBaker** [comment]
Thanks for the report @phil123456 

While Audio has seen quite a few and varied tweaks over the years since Opentoonz was released much of the code is still original as inherited from Toonz source.  It would be good to get audio experts and aficionados to chart a course for improvements moving into v1.7... v2.0 and farther ahead; your suggestion included.

Related aside:  From what I've gathered most use Opentoonz audio as a temporary scratch track pending audio replacement via dedicated audio programs.  It would be good to review how audio can best be used in Opentoonz as well as to generate final outputs well as how it best fits into pipelines with other dedicated audio programs. 

Interestingly, I was eager to test your report because I recall others reporting they could not get audio in subxsheets to even play.  As such I needed to refresh my memory that even worked.  Could what you are experiencing be a new problem based on a new  capability we didn't have before?  Not sure.

**Comment by phil123456** [comment]
you sound like nobody's in charge :-)
it's more a ui feature than complex audio btw

**Comment by RodneyBaker** [comment]
Definitely no one is in charge of this requested update as of yet... well, actually you are. 
You get to lead the way.  :)

In my response I was a bit verbose (as is my general approach) in attempting to capture the general pulse of audio features in Opentoonz.  And I certainly don't want to down play the many improvements to audio that have been made since 2016.
Your request should be fairly simple to execute but is one piece of a much larger puzzle.
This is me just trying to juggle it all in my head.  ;)

I truly was under the impression that audio files were not playing while collapsed into subxsheets.
I'm not sure where that idea originates.

I will tag this request with UI although it will be a mite more complicated than that as OT will require additional functionality under the hood to reach the goal.  Optimistically, one would think that most of the code will already be in place.

**Comment by phil123456** [comment]
let's hope
